src/main/java/useCases/generator/RevisedPropertyGenerator.java

```diff
@@ -0,0 +1,54 @@
+package useCases.generator;
+
+import entity.Property;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * It is a generator generating the property blocks
+ */
+public class RevisedPropertyGenerator {
+
+    static final String DELIMITER = ",";
```

Comment:
Do we want to use the RegEx delimiter as used in `InitController`?
This one
",(?=([^\"]*\"[^\"]*\")*[^\"]*$)"